chat-demo-android/src/main/kotlin/com/twilio/chat/demo/activities/ChannelActivity.kt

```diff
@@ -52,9 +52,9 @@ class ChannelActivity : Activity(), ChatClientListener, AnkoLogger {
         getChannels()
     }
 
-    override fun onCreateDescription(): CharSequence? {
+    override fun onDestroy() {
         basicClient.chatClient?.removeListener(this@ChannelActivity)
-        return super.onCreateDescription()
+        super.onDestroy()
```

Comment:
@emartynov While it's nice to get these demos in shape, please don't waste too much time on fixing them - they will be deleted and replaced by a properly architected app samples.

Reply:
That was my last change if you people don't request any changes more

Reply:
Ok, that's fine, just don't want you to spend too much time on this.